### ISSUE TITLE
Slightly-off Renderings

### ISSUE THREAD

**Issue by feeltheburn** [coleminor/el-misc-tools#4]
For elm2pov:

Not sure if this is an elm2pov issue, or an issue in the Games::EternalLands::Loader

The resulting images seem to be slightly-off. 

The attached image shows an example (map8_insides - Naralik magic school + houses).

The image is a direct output from elm2pov to an 1152x1152 image. (1152 being 3x the actual map's tile size of 384x384)

What I added to it: I darkened the map slightly, then overlayed it with "walkable tiles". I created a script that uses Loader and properly prints a transparent white where there is a walkable tile. 

I thought I might have gotten my scripting wrong, but on testing the map in the client, I saw my walkable tiles were correct, it was the underlying map created by elm2pov that was off.

You see the "X" marking my location in the image. At that spot, I am sitting on the very right side of the bench which you can see directly above the X. 

Basically, the resulting output of elm2pov seems to be a bit "too high and too right" for its actual location.

This is only noticeable on small maps. The larger the map, the less noticeable. (Whitestone - map2 - for example, can't see it.)

The C1 Portal Room also is a good example to use for testing.

![xmarksthewrongspot](https://f.cloud.github.com/assets/2794779/2428532/12f54ad4-ac4c-11e3-8877-9d69c7f81bba.png)
